src/components/button/button.js

```diff
@@ -11,6 +11,8 @@
 import '../../vendor/polyfills/Event' // addEventListener and event.target normaliziation
 
 var KEY_SPACE = 32
+var DEBOUNCE_TIMEOUT_IN_SECONDS = 1
+var debounceFormSubmitTimer = null
```

Comment:
Is there anything weird that could happen here if you had two forms / submit buttons on one page? Would it be better to scope this to the submitted form somehow?

Reply:
While there's a single timer for all buttons I don't think it's humanly possible to click two buttons quick enough for it to matter?

🏃